info/token-info-storage-enq.json

```diff
@@ -1,107 +1,21 @@
 [
     {
-        "token_id":"0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
@DimitriiVM  You have deleted ENQ token description, as some other legit tokens. Please bring them back. I think it will be easier to start from scratch. Add you token, do not delete existing tokens, or template.